.travis.yml

```diff
@@ -16,6 +16,12 @@ matrix:
       env: TOXENV=django31-drf311 AFTER_SUCCESS=codecov
     - python: '3.8'
       env: TOXENV=django32-drf311 AFTER_SUCCESS=codecov
+    - python: '3.8'
+      env:
+        - COMMAND="make quality"
+        - PIPEXTRA="-r requirements/quality.txt"
+        - DEPLOY_PYPI=true
```

Comment:
we don't need this variable since there's a separate Github workflow handling PyPi deployments